### DISCUSSION TITLE
what is the gradle replacement to maven

### DISCUSSION THREAD

**Discussion by sagarbandamwar** [afreakyelf/Pdf-Viewer#62] [General]
I am adding gradle libraries instead maven 

allprojects {
  repositories {
    ...
    maven { url "https://jitpack.io" }
  }
}

so what is the replacement for this?

**Comment by afreakyelf** [comment]
I am not sure what do you mean. As of now this is the only way this library can be used, however i am working on to get it released on mavencentral
